libsolc/libsolc.h

```diff
@@ -42,9 +43,11 @@ extern "C" {
 /// @param o_contents A pointer to the contents of the file, if found.
 /// @param o_error A pointer to an error message, if there is one.
 ///
-/// If the callback is not supported, o_contents and o_error should be set to NULL.
+/// The file contents that is to be allocated by the callback implementor must use the
```

Comment:
Should make this explicit that both `o_contents` and `o_error` must follow the same rules.